openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -119,7 +119,7 @@ def is_course_cohorted(course_key):
     Raises:
        Http404 if the course doesn't exist.
     """
-    return _get_course_cohort_settings(course_key).is_cohorted
+    return get_course_cohort_settings(course_key).is_cohorted
```

Comment:
@staubina I intentionally made this method private because I don't want people looking at CourseCohortSettings, beyond the "is_cohorted" flag (to avoid confusion with the deprecated discussion settings contained within). There are public setters and getters for that property-- is_course_cohorted and set_course_cohorted.